src/types/generatedGraphQLTypes.ts

```diff
@@ -2460,7 +2453,6 @@ export type Resolvers<ContextType = any> = {
   Time?: GraphQLScalarType;
   Translation?: TranslationResolvers<ContextType>;
   URL?: GraphQLScalarType;
-  UpdateUserTagInput?: UpdateUserTagInputResolvers<ContextType>;
   Upload?: GraphQLScalarType;
```

Comment:
Why we need these changes in this PR?

Reply:
In this whole file ?

Reply:
@noman2002 I didn’t make that change.
It probably got changed when I made the commit : -


https://user-images.githubusercontent.com/92572013/225726297-2c545b1e-37d0-4559-89ba-fefda465c58c.mp4

<br>
Should I undo those changes and make a new commit?

Reply:
yes please